scripts/build.py

```diff
@@ -2009,6 +2012,7 @@ def _form_env(self):
                       '%s/build_tools' % self.defold,
                       '%s/ext/lib/python' % self.dynamo_home]
         env['PYTHONPATH'] = os.path.pathsep.join(pythonpaths)
+        env['PYTHONIOENCODING'] = 'UTF-8'
```

Comment:
I don't actually remember why this was needed. Can be related to older environment where I used msys.